### DISCUSSION TITLE
[MOCK IDEA] two-factor authentication for svn commits

### DISCUSSION THREAD

**Discussion by Humbedooh** [apache/infrastructure-ideas#2] [Enhancements]
This is a mock suggestion for adding a layer of two-factor authentication to subversion commits.
Comments and upvotes from others could be used to signify support for the adaptation of this idea.

**Comment by cottage14** [comment]
How many projects would that affect?

**Reply by Humbedooh** [reply]
all projects, essentially.
everyone uses svn for releases right now.

**Comment by dave2wave** [comment]
MacOS svn already wants my fingerprint to send my password .... is that enough?
